draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -384,25 +390,22 @@ and are less efficient than hash and encode methods.
 
 # Algorithm Recommendations {#recommendations}
 
-The following table lists algorithms recommended by use-case:
-
-| Application       | Requirement   | Additional Details
-|-------------------|---------------|---------|
-| SPEKE {{Jablon96}}| Naive         | H(x)*G |
-| PAKE  {{BMP00}}   | Random Oracle |   -    |
-| BLS {{BLS01}}     | Random Oracle            |    -   |
-| IBE {{BF01}}      | Random Oracle | Supersingular, pairing-friendly curve |
-| PRF | Injective encoding | F(k, m) = k*H(m) |
+In practice, two types of mappings are common: (1) Injective encodings, as can be used to
+construct a PRF as F(k, m) = k\*H(m), and (2) Random Oracles, as required by PAKEs {{BMP00}},
+BLS {{BLS01}}, and IBE {{BF01}}. (Some applications, such as IBE, have additional requirements,
+such as a Supersingular, pairing-friendly curve.)
 
-To find the suitable algorithm, lookup the requirement from above, with
-the chosen curve in the below:
+The following table lists recommended algorithms for different curves and mappings. To select
+a suitable algorithm, choose the mapping associated with the target curve. For example,
+Elligator2 is the recommended injective encoding function for Curve25519, whereas Simple SWU
+is the recommended injective encoding for P-256.
```

Comment:
Maybe add a sentence to explain the FFSTV(_) notation?

Reply:
Yep, will do!